npm/angular/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Ooooh this is going to be controversial !!

Reply:
Removed, let's avoid controversy for now :)